### ISSUE TITLE
Mixed boolean operators without parentheses

### ISSUE THREAD

**Issue by TimWolla** [WoltLab/WCF#3926]
```
FILE: wcfsetup/install/files/lib/system/form/builder/field/TI18nFormField.class.php
-----------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------
 424 | ERROR | Mixed '&&' and '||' within an expression without using parentheses.
-----------------------------------------------------------------------------------


FILE: wcfsetup/install/files/lib/system/html/input/node/HtmlInputNodeWoltlabMetacodeMarker.class.php
----------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------
 385 | ERROR | Mixed '&&' and '||' within an expression without using parentheses.
----------------------------------------------------------------------------------------------------


FILE: wcfsetup/install/files/lib/system/language/I18nPlural.class.php
-----------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------
 385 | ERROR | Mixed '&&' and '||' within an expression without using parentheses.
-----------------------------------------------------------------------------------


FILE: wcfsetup/install/files/lib/system/option/user/group/BooleanUserGroupOptionType.class.php
----------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------
 60 | ERROR | Mixed '&&' and '||' within an expression without using parentheses.
----------------------------------------------------------------------------------------------


FILE: wcfsetup/install/files/lib/system/worker/SitemapRebuildWorker.class.php
-----------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------
 154 | ERROR | Mixed '&&' and '||' within an expression without using parentheses.
-----------------------------------------------------------------------------------
```

**Comment by TimWolla** [comment]
@Gravatronics Please check TI18nFormField. I am not able to determine which grouping is correct there.
@dtdesign Please check HtmlInputNodeWoltlabMetacodeMarker. I believe the parentheses should be around the `||` which would make the current version a bug.
@joshuaruesweg Please check SitemapRebuildWorker. I am not able to determine which grouping is correct there.

**Comment by dtdesign** [comment]
Fixed in 8f6eebb994d9d9687d30aa1e4e1ea2ea0f5ef2d0

**Comment by joshuaruesweg** [comment]
Fixed the `SitemapRebuildWorker` in 312e6efadd9bfc2dfd92007e8c7e773515191b16

**Comment by TimWolla** [comment]
All resolved now.
